app/editor/src/features/content/form/ContentSummaryForm.tsx

```diff
@@ -79,241 +87,273 @@ export const ContentSummaryForm: React.FC<IContentSummaryFormProps> = ({
-              setFieldValue('timeTrackings', [
-                ...values.timeTrackings,
-                {
-                  userId: userId ?? 0,
-                  activity: !!values.id ? 'Updated' : 'Created',
-                  effort: (values as any).prep,
-                  createdOn: new Date(),
-                },
-              ]);
-              setFieldValue('prep', '');
-            }}
-          >
-            Add
-          </Button>
+        <Row>
           <FormikText
             disabled
-            className="sm"
-            name="total"
-            label="Total"
-            value={effort?.toString()}
+            width={FieldSize.Large}
+            name="tags"
+            label="Tags"
+            value={values.tags.map((t) => t.id).join(', ')}
           />
           <Button
+            variant={ButtonVariant.danger}
+            className="top-spacer"
             onClick={() => {
-              setContent({ ...content, timeTrackings: values.timeTrackings });
-              toggle();
+              const regex = /\[.*\]/; // TODO: This is far too eager and could remove value content.
+              setFieldValue('summary', values.summary.replace(regex, ''));
+              setFieldValue('tags', []);
             }}
-            style={{ marginTop: '1.16em' }}
-            variant={ButtonVariant.secondary}
           >
-            View Log
+            Clear Tags
           </Button>
-          <Modal
-            hide={toggle}
-            isShowing={isShowing}
-            headerText="Prep Time Log"
-            body={<TimeLogTable totalTime={effort} data={values.timeTrackings} />}
-            customButtons={
-              <Button variant={ButtonVariant.secondary} onClick={toggle}>
-                Close
-              </Button>
-            }
+        </Row>
+        <Row className="row-margins">
+          <FormikRadioGroup
+            label="Toning"
+            direction="row"
+            name="tonePool"
+            required
+            options={toningOptions}
+            onChange={(e, value) => {
+              setFieldValue('tonePool', value);
+              setFieldValue('tone', value?.value);
+            }}
           />
         </Row>
-      </Show>
-    </Col>
+        <Show visible={contentType === ContentType.Snippet}>
+          <Row className="row-margins">
+            <Upload
+              id="upload"
+              name="file"
+              file={file}
+              onSelect={(e) => {
+                const file = !!e.target?.files?.length ? e.target.files[0] : undefined;
+                setFieldValue('file', file);
+                // Remove file reference.
+                setFieldValue('fileReferences', []);
+              }}
+              onDownload={() => {
+                download(values.id, file?.name ?? `${values.source}-${values.id}`);
+              }}
+              onPostDelete={() => {
+                setStreamUrl('');
+                if (!!videoRef.current) {
```

Comment:
You can rewrite this `videoRef.current?.src = '';`

Reply:
This gave me:

The left-hand side of an assignment expression may not be an optional property access.

Reply:
This article says I should use my original approach: https://bobbyhadz.com/blog/typescript-left-hand-side-of-assignment-not-optional

Reply:
I think you're correct.